apps/HelloBaremetal/cmake/toolchain.noos-arm32-sample.cmake

```diff
@@ -0,0 +1,25 @@
+# CMake toolchain setup for AArch32 baremetal target
+set(CROSS_PREFIX "arm-none-eabi-")
+set(CMAKE_CROSSCOMPILING ON)
```

Comment:
Can you not `set(CMAKE_SYSTEM_NAME none)` here instead? Normally one does not set `CMAKE_CROSSCOMPILING` directly. But more to the point, I think that not setting `CMAKE_SYSTEM_NAME` will default it to the host system (e.g. Linux), which is probably not what we want.

Reply:
Fixed, thanks.